documentation/guides/api/quickstart.md

```diff
@@ -0,0 +1,96 @@
+# Quickstart guide
+
+This is the quick start guide for setting up and running the API locally.
+
+## Prerequisites
+
+Refer to the [general setup guide](../general_setup.md) for setting up the
+prerequisites.
```

Comment:
Added a link to the matrix in 840633b. I didn't add the bullet points because it would be duplicate info that was very likely to go out of sync between the two places.

Reply:
Great point, even better 💯